inst/examples/sir.cpp

```diff
@@ -62,7 +62,9 @@ sir::init_t dust_data<sir>(cpp11::list data) {
   double R0 = 0.0;
 
   // Rates, which can be set based on the provided data
+  // [[dust::param(beta, required = FALSE)]]
```

Comment:
This model doesn't also have the dust::model and dust::class descriptors, and they are still picked up automatically? Do we have a test somewhere for those (and that they can be different)?